tests/assemblers/test_xgboost.py

```diff
@@ -228,3 +228,47 @@ def test_multi_class_best_ntree_limit():
     ])
 
     assert utils.cmp_exprs(actual, expected)
+
+
+def test_regression_saved_without_feature_names():
+    import os
+
+    base_score = 0.6
+    estimator = xgboost.XGBRegressor(n_estimators=2, random_state=1,
+                                     max_depth=1, base_score=base_score)
+    utils.train_model_regression(estimator)
+
+    filename = "tmp.file"
+    if os.path.exists(filename):
```

Comment:
Thanks for adding the test! 
May I please ask you to use `tmp_dir` utility function to avoid polluting user's workspace. Example can be found here:
https://github.com/BayesWitnesses/m2cgen/blob/00abbff1e978536a642355932a1a9c121feea8ca/tests/e2e/executors/base.py#L12
Note that this context manager will clean up the temporary directory automatically so you don't have to do it manually here. 
I'd also suggest to use UUID for a filename in place of hardcoded `tmp.file` to avoid potential conflicts, since this particular test should not be affected by artifacts which can potentially be produced by other tests.

Great job otherwise 👍